examples/webgl_loader_pdb.html

```diff
@@ -210,7 +210,7 @@
 						start.multiplyScalar( 75 );
 						end.multiplyScalar( 75 );
 
-						const object = new THREE.Mesh( boxGeometry, new THREE.MeshPhongMaterial( 0xffffff ) );
+						const object = new THREE.Mesh( boxGeometry, new THREE.MeshPhongMaterial( { color: 0xffffff } ) );
```

Comment:
The `MeshPhongMaterial` takes an object of parameters. `0xffffff` matches the default color, so alternatively it could be removed altogether.